content/pages/startComputeDataset.json

```diff
@@ -18,6 +18,11 @@
         "options": [],
         "sortOptions": false
       }
-    ]
+    ],
+    "termsAndConditions": {
+      "name": "termsAndConditions",
+      "options": ["Terms and Conditions"],
+      "prefixes": ["I agree to the"]
+    }
```

Comment:
issue/question: the second checkbox seems to be missing here, is the content located somewhere else?